src/main/java/step3/domain/LottoNumber.java

```diff
@@ -0,0 +1,53 @@
+package step3.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+	private final int number;
+
+	public static final int MIN_LOTTO_VALUE = 1;
+	public static final int MAX_LOTTO_VALUE = 45;
+
+	private static final String LOTTO_NUMBER_PHRASES = "로또 당첨 번호는 1 이상 45 이하의 숫자여야 합니다.";
+
+	public LottoNumber(int number) {
+		validLottoNumber(number);
+		this.number = number;
+	}
+
+	private void validLottoNumber(int number) {
+		if (number < MIN_LOTTO_VALUE) {
```

Comment:
```suggestion
		if (number < MIN_LOTTO_VALUE || number > MAX_LOTTO_VALUE) {
```
동일한 예외를 제공한다면 코드를 줄여볼수 있을것 같아요.